File: CodeChef/Beginner/09ReverseNumber/reversenum.java
import java.util.Scanner;

class reversenum {

    public static int Reverse(int x) {
        String st = String.valueOf(x);
        String ans = "";
        for (int i = st.length() - 1; i >= 0; i--) {
            ans += st.charAt(i);
        }

        return Integer.valueOf(ans);
    }

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int n, x;

        n = sc.nextInt();
        for (int i = 0; i < n; i++) {
            x = sc.nextInt();
            System.out.println(Reverse(x));
        }
        sc.close();
    }
}

File: CodeChef/Beginner/09ReverseNumber/reversenum.cpp
#include <iostream>
using namespace std;

class reversenum {
public:
  int reverse(int x) {
    string st = to_string(x);
    string ans = "";

    for (int i = st.size() - 1; i >= 0; i--) {
      ans += st[i];
    }
    return stoi(ans);
  }
};
int main() {
  int n, x;
  cin >> n;
  reversenum rnobj = reversenum();
  for (int i = 0; i < n; i++) {
    cin >> x;
    cout << rnobj.reverse(x) << endl;
  }

  return 0;
}